packages/components/src/components/navigation-item/navigation-item.stories.ts

```diff
@@ -1,464 +1,161 @@
-            'divider',
-            'current-indicator',
-            'details'
-          ].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-navigation-item::part(${part}){outline: solid 2px red}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [
-        {
-          type: 'attribute',
-          name: 'vertical',
-          value: true
-        },
-        {
-          type: 'attribute',
-          name: 'divider',
-          value: true
-        },
-        {
-          type: 'attribute',
-          name: 'chevron',
-          value: true
-        },
-        {
-          type: 'slot',
-          name: 'icon-right',
-          value: '<sd-icon library="global-resources" name="system/picture" slot="icon-right"></sd-icon>'
-        },
-        {
-          type: 'slot',
-          name: 'icon-left',
-          value: '<sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>'
-        },
-        {
-          type: 'slot',
-          name: 'main',
-          value: '<sd-badge slot="main">888</sd-badge>'
-        },
-        {
-          type: 'slot',
-          name: 'children',
-          value: '<div slot="children" class="slot slot--border slot--background h-6"></div>'
-        },
-        descriptionSlotConstant
-      ],
-      args
-    });
-  }
+export const Description = {
+  render: () =>
+    html` <sd-navigation-item vertical>
+      Vertical Navigation with Description
+      <p slot="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullam.</p>
+    </sd-navigation-item>`
 };
 
 /**
- * `sd-navigation-item` is fully accessibile via keyboard in all variants. Tab through the story to try each variant.
+ * Use `indented` attribute to add padding to the left side.
+ * - Only works with `vertical` attribute.
  */
 
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">
-      ${['Button', 'Link', 'Accordion'].map(title => {
-        const constants: ConstantDefinition[] = [
-          { type: 'attribute', name: 'vertical', value: true },
-          { type: 'attribute', name: 'chevron', value: true },
-          {
-            type: 'slot',
-            name: 'default',
-            value: '<div style="width: 245px; text-align: left;">Navigation</div>'
-          }
-        ];
-
-        if (title === 'Link')
-          constants.push({ type: 'attribute', name: 'href', value: 'https://www.union-investment.de/' });
-        if (title === 'Accordion') constants.push(childrenSlotConstant);
-
-        return generateTemplate({
-          args,
-          constants,
-          options: { title }
-        });
-      })}
-    </div>`;
-  },
+export const Indented = {
+  render: () =>
+    html`<div class="w-[400px]">
+      <sd-navigation-item vertical indented divider>Indented Navigation</sd-navigation-item>
```

Comment:
Looks different to [figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=3051-15845&node-type=FRAME&t=23EIrSTNYEo83Lxr-11)?

![image](https://github.com/user-attachments/assets/1ce77b69-15a2-4e40-a863-3f2dd922817b)